ocs_ci/ocs/resources/pod.py

```diff
@@ -945,6 +946,22 @@ def get_pvc_name(pod_obj):
     ).get('volumes')[0].get('persistentVolumeClaim').get('claimName')
 
 
+def get_pvc_obj(pod_obj):
+    """
+    Function to get pvc_name from pod_obj
```

Comment:
Function to get pvc object from pod object

Reply:
fixed

Reply:
still pvc_name instead of PVC object :)

Reply:
pushed a new commit just now